app/window.go

```diff
@@ -464,13 +464,31 @@ const (
 
 // WindowMode sets the window mode.
 //
-// Supported platforms are macOS, X11 and Windows.
+// Supported platforms are macOS, X11, Windows and JS.
 func WindowMode(mode wm.WindowMode) Option {
 	return func(opts *wm.Options) {
 		opts.WindowMode = &mode
 	}
 }
 
+const (
+	// UnlockedOrientation is the default behavior, allowing the rotate of the screen.
+	UnlockedOrientation = wm.UnlockedOrientation
+	// LockedPortrait locks the screen to portrait.
+	LockedPortrait = wm.LockedPortrait
+	// LockedLandscape locks the screen to landscape.
+	LockedLandscape = wm.LockedLandscape
+)
```

Comment:
I don't know if the names are right...

I choose `UnlockedOrientation` instead of `Any`, because `app.Any` seems wrong, since it doesn't means anything. Maybe `AnyOrientation` ? The `Locked` and `Unlocked` is to make obivous that the screen can/can't rotate.

Reply:
Every device (at least mobile devices) has "2x Portrait" and "2x Landscape". I mean, you can hold your smartphone at portrait mode or in a portrait mode upside down (180º rotated from the stardand portrait).

Maybe it could be extended to:

- UnlockedOrientation
- LockedPortrait               // Allow the rotation in 180º (primary or upside down; the sensor of the device can still rotate).
- LockedPrimaryPortrait   // Lock into the "primary" portrait.
- LockedReversedPortrait // Lock into the "reversed portrait (upside down).
- LockedLandscape
- LockedPrimaryLandscape
- LockedReversedLandscape

Currently the `LockedPortrait` and `LockedLandscape` behaves as `LockedPrimaryPortrait` and `LockedPrimaryLandscape`.

-----------

However, I think it's better to keep it simple and extend it once iOS (and maybe others OSes) gets supported. :)

Reply:
Thanks for working on this. I think "locked" is too similar to "lock screen". How about:

```
        // AnyOrientation allows the window to be freely orientated.
	AnyOrientation
	// PortraitOrientation constrains the window to portrait orientations.
	PortraitOrientation
	// LandscapeOrientation constrains the window to landscape orientations.
	LandscapeOrientation
```